NewHorizons/Handlers/PlanetCreationHandler.cs

```diff
@@ -1,629 +1,718 @@
-using NewHorizons.Builder.Atmosphere;
```

Comment:
Holy line endings batman

fr though these line endings should be fixed so we can review this file properly